plone/app/z3cform/configure.zcml

```diff
@@ -39,13 +39,6 @@
       class=".templates.RenderContentProvider"
       permission="zope.Public"
       />
-  <browser:page
-      name="ploneform-render-widget"
-      for="z3c.form.interfaces.ISingleCheckBoxWidget"
-      layer="plone.app.z3cform.interfaces.IPloneFormLayer"
-      class=".templates.RenderSingleCheckboxWidget"
-      permission="zope.Public"
-      />
```

Comment:
youre right, know, that we have an own widget for this, I can provide an own display template for this case in order to show the description right. 
@agitator may you provide me with the markup needed for this?